test/hotspot/jtreg/runtime/ParallelLoad/MyLoader.java

```diff
@@ -61,6 +61,7 @@ public Class loadClass(String name) throws ClassNotFoundException {
     private static boolean parallel = false;
     private Object sync = new Object();
     private Object thread_sync = new Object();
+    private static volatile boolean ready = false;
```

Comment:
The name "waiting" would be more appropriate.

Reply:
Ok.